zerver/views/events_register.py

```diff
@@ -121,7 +99,7 @@ def events_register_backend(
         include_streams = False
 
     if client_capabilities is None:
-        client_capabilities = {}
+        client_capabilities = ClientCapabilities(notification_settings_null=False)
```

Comment:
How did you check that `False` is the correct value to use here? I guess I notice we treat it not being present as `False`, so maybe that's correct?

`notification_settings_null = client_capabilities.get("notification_settings_null", False)`

Reply:
> `notification_settings_null = client_capabilities.get("notification_settings_null", False)`
Yes, this is one of the first things the downstream function does, so essentially not being present is the same as false and since its a required `key` it had to be specified.